src/custom-element.js

```diff
@@ -1640,6 +1645,19 @@ function createBaseCustomElementClass(win) {
 
     /**
      * Returns an optional overflow element for this custom element.
+     * @return {!./layout-delay-meter.LayoutDelayMeter}
+     */
+    getLayoutDelayMeter_() {
+      if (!this.layoutDelayMeter_) {
```

Comment:
As discussed offline,
- we sample page views using `perf.isPerformanceTrackingOn()`
- we apply a rate limit of 10 per second for the CSI pings

PTAL